packages/bridge-ui-v2/src/components/Transactions/StatusFilterDropdown.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+  import { t } from 'svelte-i18n';
+
+  import IconFlipper from '$components/Icon/IconFlipper.svelte';
+  import { MessageStatus } from '$libs/bridge';
+  import { classNames } from '$libs/util/classNames';
+  import { closeOnEscapeOrOutsideClick } from '$libs/util/customActions';
+
+  export let selectedStatus: MessageStatus | null = null;
+
+  let flipped = false;
+
+  let iconFlipperComponent: IconFlipper;
+
+  let menuOpen = false;
+
+  const options = [
+    { value: null, label: $t('transactions.filter.all') },
+    { value: MessageStatus.NEW, label: $t('transactions.filter.processing') },
+    { value: MessageStatus.RETRIABLE, label: $t('transactions.filter.retry') },
```

Comment:
When a transaction is retriable?

Reply:
When a previous claim (processMessage) failed for whatever reason.

Have a look at the bridge README and contract!

- [Relevant contract part](https://github.com/taikoxyz/taiko-mono/blob/92392b26030a010943433f04b22b39b18d3f218f/packages/protocol/contracts/bridge/libs/LibBridgeProcess.sol#L124)
-  [README](https://github.com/taikoxyz/taiko-mono/blob/92392b26030a010943433f04b22b39b18d3f218f/packages/protocol/contracts/bridge/README.md?plain=1#L100)